File: CS/LegendTitleSample/Form1.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace LegendTitleSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            #region #LegendTitleOptions
            // Specify legend title's text.
            chart.Legend.Title.Text = "Area of countries";
            // Show legend title.
            chart.Legend.Title.Visible = true;
            // Configure legend title appearance.
            chart.Legend.Title.Alignment = StringAlignment.Near;
            chart.Legend.Title.MaxLineCount = 3;
            chart.Legend.Title.WordWrap = true;
            #endregion #LegendTitleOptions
        }
    }
}

File: VB/LegendTitleSample/Form1.vb
Imports System
Imports System.Drawing
Imports System.Windows.Forms

Namespace LegendTitleSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
'            #Region "#LegendTitleOptions"
            ' Specify legend title's text.
            chart.Legend.Title.Text = "Area of countries"
            ' Show legend title.
            chart.Legend.Title.Visible = True
            ' Configure legend title appearance.
            chart.Legend.Title.Alignment = StringAlignment.Near
            chart.Legend.Title.MaxLineCount = 3
            chart.Legend.Title.WordWrap = True
'            #End Region ' #LegendTitleOptions
        End Sub
    End Class
End Namespace
